name.abuchen.portfolio.ui/src/name/abuchen/portfolio/ui/views/dataseries/DataSeries.java

```diff
@@ -124,6 +163,9 @@ public Object getInstance()
 
     public String getLabel()
     {
+        if (instance instanceof GroupedDataSeries c && groups.length > 0)
+            return groups[groups.length - 1] + " - " + label; //$NON-NLS-1$
+
```

Comment:
This PR is using the "group" essentially as second label.
I understand that we need two labels (in the selection dialog the full label does not make sense, in the chart legend it is required to understand the data series).
I think we should make it then explicit and not the logic that the last element in the object array is a label.